### DISCUSSION TITLE
What's after Authorization V1?

### DISCUSSION THREAD

**Discussion by IgniteMosquito** [amzn/selling-partner-api-models#3716] [General]
Hi Team,

It might be a silly question, but I just noticed that Authorization v1 is depreciated (https://developer-docs.amazon.com/sp-api/docs/sp-api-deprecations). What is next then? 
Because the documentation is only for v1: https://developer-docs.amazon.com/sp-api/docs/authorization-api-v1-reference

Thanks,
Basia

**Comment by MeenaAmz** [comment]
The main use of authorization API was to migrate an Amazon Marketplace Web Service (Amazon MWS) authorization, that a seller previously granted a developer, to a hybrid Selling Partner API (SP-API) application. This eliminates the need to request authorization from this seller for hybrid SP-API application. So this was more an interim solution to aid the third party(3P) developers with faster migration to SP-APIs without disrupting business. As specified in the [doc](https://developer-docs.amazon.com/sp-api/docs/authorization-api-use-case-guide), "Authorizations of a hybrid SP-API application expire after one year. Be sure to have your selling partners reauthorize your SP-API hybrid application yearly before their authorizations expire." So there will not be a v2 for authorizations API, 3P developers are expected to have their selling partners reauthorized by that time.
